pom.xml

```diff
@@ -974,8 +984,8 @@
   </dependency>
   <dependency>
     <groupId>com.github.sarxos</groupId>
-    <artifactId>webcam-capture</artifactId>
-    <version>0.3.10</version>
+    <artifactId>webcam-capture-driver-v4l4j</artifactId>
+    <version>0.3.13-SNAPSHOT</version>

```

Comment:
Why are we using a snapshot build here?  is there something in 0.3.13-SNAPSHOT specifically that we need compared to 0.3.12 ?

Reply:
its the only version that works on all other platforms "and" raspi